sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/Language.cs

```diff
@@ -29,6 +29,7 @@ public Language(string value)
         private const string ItValue = "it";
         private const string NlValue = "nl";
         private const string PtValue = "pt";
+        private const string JaValue = "ja";
```

Comment:
Just out of curiosity, is this Japanese?

Reply:
It is!